SwiftWisdomTests/StandardLibrary/ArrayUtilitiesTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Array+UtilitiesTests.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 9/30/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+import SwiftiWisdom
+
+class ArrayUtilities: XCTestCase {
+
+    func testSubArrayFromIndicesOf() {
+        let testArray = [4, 8, 15, 16, 23, 42]
+        let indices = testArray.ip_indicesOf(elements: [15, 16, 19])
```

Comment:
This test is dependent on `ip_indicesOf`; it could fail if `ip_indicesOf` doesn't do what you intend. So, I think you should hand-code the indices you're looking for.  

You can just do `let indices = [2, 3]` or even `.. = [2, 3, 20]` if you wanna still wanna see if you can trick it like you kinda are with that `19` in `ip_indicesOf`.